### ISSUE TITLE
[SF][UI5-Popover][BUG][Urgent][0.31.19] accessibleName doesn't work 

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#4005]
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.
we use NVDA to test a11y issues in our project

**The version of ui5-webcomponent is 0.31.19**

**Reproduce Step:**
1. open https://github.wdf.sap.corp/pages/xweb/common-react/?path=/story/react-components-first-time-guidance--default
2. click "show the first normal slide"
3. click the "start tour" button

![Screen Shot 2021-09-27 at 2 21 44 PM](https://user-images.githubusercontent.com/78520316/134876837-1d2d4969-2241-4f24-af6c-375520de4c16.png)
![Screen Shot 2021-09-27 at 5 00 17 PM](https://user-images.githubusercontent.com/78520316/134877733-0fa99f89-cf35-49ae-8adf-7a6ebf0e894e.png)

We can see that:

**The accessiblename is "test accessibleName attribute for popover", but this text is not announced by screen reader**

- Organization: SF

**Comment by Todor-ads** [comment]
Hi @SAP/ui5-webcomponents-topic-rd,

The Popover component is part of your topic.

Best regards,
Todor

**Comment by dimovpetar** [comment]
Hello @sfsf-xweb-sh ,

In release 0.31 the property is called `ariaLabel`. It has been renamed to `accessibleName` in the newer versions with #3449. Please use the `ariaLabel` (`aria-label` as attribute) in your case instead.

Best regards,
Petar

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh 
we introduced `accessibleName` for the Popover component
and it will be available with 0.31.20.

I am keeping the issue open, so you can close the issue yourself after trying it out.

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh the 0.31.20 is released and accessibleName has been added to the Popover.
